src/Meziantou.Framework.Win32.Jobs/JobObject.cs

```diff
@@ -75,6 +75,28 @@ public static JobObject Open(JobObjectAccessRights desiredAccess, bool inherited
         return new JobObject(handle);
     }
 
+    // Win32 error code
+    private const int ERROR_FILE_NOT_FOUND = 2;
```

Comment:
I think you can move this to https://github.com/meziantou/Meziantou.Framework/blob/main/src/Meziantou.Framework.Win32.Jobs/NativeMethods.txt. The source generator will generate the constant name and value.